src/App.tsx

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Dashboard from './dashboard/Dashboard';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Bank from './layout/Bank/Bank';
+import MasterLayout from './layout/Home/Home';
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<MasterLayout />} />
+          <Route path="/bank" element={<Bank />} />
```

Comment:
please remove this route since its not used, u replace it with the one inside layout component